model/trace/rpc.yaml

```diff
@@ -89,32 +89,14 @@ groups:
           RPC conventions for more information.
 
   - id: rpc.message
-    prefix: "message"  # TODO: Change the prefix to rpc.message?
+    prefix: rpc.message
     type: event
     brief: "RPC received/sent message."
     attributes:
-      - id: type
-        type:
-          members:
-            - id: sent
-              value: "SENT"
-            - id: received
-              value: "RECEIVED"
-        stability: experimental
-        brief: "Whether this is a received or sent message."
-      - id: id
-        type: int
-        stability: experimental
-        brief: "MUST be calculated as two different counters starting from `1` one for sent messages and one for received message."
-        note: "This way we guarantee that the values will be consistent between different implementations."
-      - id: compressed_size
-        type: int
-        stability: experimental
-        brief: "Compressed size of the message in bytes."
-      - id: uncompressed_size
-        type: int
-        stability: experimental
-        brief: "Uncompressed size of the message in bytes."
+      - ref: rpc.message.type
```

Comment:
Let's add requirement level here explicitly

Reply:
I have added it everywhere and will be updating it for further PRs as well